vision/getting-started/image_editing.ipynb

```diff
@@ -1,710 +1,1247 @@
 {
```

Comment:
<p>"... expanded is not too large".</p><p> </p><p>What is defined as "not too large"? We should be more specific here, or alternatively provide looser guidance as in: "outpainting generally works better when there is less area to outpaint than if there is more."</p><p> </p><p>Same with "relatively smooth/not gritty"</p><p>is not precise language and may confuse developers as to what is meant to "relatively smooth/not gritty". It may be good to show visual examples.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/506/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Reply:
Updated description